src/stepFunctions/commands/visualizeStateMachine/abstractAslVisualizationManager.ts

```diff
@@ -0,0 +1,40 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as nls from 'vscode-nls'
+
+import { Logger } from '../../../shared/logger'
+
+const localize = nls.loadMessageBundle()
+
+export abstract class AbstractAslVisualizationManager {
+    private readonly extensionContext: vscode.ExtensionContext
+
+    public constructor(extensionContext: vscode.ExtensionContext) {
```

Comment:
Nit: Merge `extensionContext` variable into parameter since it isn't modified in constructor.